examples/templates/kubernetes/README.md

```diff
@@ -7,7 +7,32 @@ icon: /icon/k8s.png
 
 # Getting started
 
-This template creates a pod running the `codercom/enterprise-base:ubuntu` image.
+This template creates a deplyment running the `codercom/enterprise-base:ubuntu` image.
+
+## Prerequisites
+
+This template uses [`kubernetes_deployment`](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/deployment) terraform resource, which requires the `coder` service account to have permission to create deploymnets. For example if you are using [helm](https://coder.com/docs/v2/latest/install/kubernetes#install-coder-with-helm) to install Coder, you should set `coder.serviceAccount.enableDeployments=true` in your `values.yaml`
+
+```diff
+coder:
+serviceAccount:
+    # coder.serviceAccount.workspacePerms -- Whether or not to grant the coder
+    # service account permissions to manage workspaces. This includes
+    # permission to manage pods and persistent volume claims in the deployment
+    # namespace.
+    #
+    # It is recommended to keep this on if you are using Kubernetes templates
+    # within Coder.
+    workspacePerms: true
+    # coder.serviceAccount.enableDeployments -- Provides the service account permission
+    # to manage Kubernetes deployments.
+-   enableDeployments: false
++   enableDeployments: true
+    # coder.serviceAccount.annotations -- The Coder service account annotations.
+    annotations: {}
+    # coder.serviceAccount.name -- The service account name
+    name: coder
+```
```

Comment:
Ah, I appreciate you changing the example template too. @ericpaulsen and I discussed changing the default Helm chart to `enableDeployments: true` and labeling it as an important change in release notes. That way, there are no extra steps.

@ericpaulsen would you be able to help with the Helm aspect of this and push to the PR?

Reply:
Sounds fine.